src/ram/transform/MakeIndex.cpp

```diff
@@ -288,6 +307,7 @@ Own<Condition> MakeIndexTransformer::constructPattern(const std::vector<std::str
             // don't permit multiple inequalities
             // TODO: @SamArch27 invariant that we have at most one indexed inequality per relation
             if (firstConstraint && inequality && seenInequality) {
+                // addCondition(Own<Condition>(clone(cond)));
```

Comment:
I think this can be removed - the move from a vector that isn't re-used should be safe.